File: quantum-portal-angular/src/app/databaseServices/ProfessionalQualificationDTO.ts
import { UserNoticeInfoDTO } from "./NoticeInfoDTO";
import { TechDTO } from "./TechDTO";

export interface ProfessionalQualificationDTO {
    isExperienced: boolean;
    years_exp: number | null;
    current_ctc: number | null;
    expected_ctc: number | null;
    expertise_tech?: TechDTO[] | null;
    familiar_tech?: TechDTO[];
    other_familiar_tech?: string | null;
    other_expertise_tech?: string | null;
    noticeInfo: UserNoticeInfoDTO;
}

File: Quantum-backend-apis/Quantum-backend-apis/DTO/ProfessionalQualificationDTO.cs
using System;
namespace Quantum_backend_apis.DTO
{
	public class ProfessionalQualificationDTO
	{
        public bool isExperienced { get; set; }
        public int? years_exp { get; set; }
        public int? current_ctc { get; set; }
        public int? expected_ctc { get; set; }
        public List<TechDTO>? expertise_tech { get; set; }
        public List<TechDTO>? familiar_tech { get; set; }
        public string? other_familiar_tech { get; set; }
        public string? other_expertise_tech { get; set; }
        public UserNoticeInfoDTO noticeInfo { get; set; }
    }
}

